packages/viz/src/modules/print/BasicsPrintView.js

```diff
@@ -63,8 +62,8 @@ class BasicsPrintView extends PrintView {
       [SITE_CHANGE_TUBING]: Images.siteChangeTubingImage
     }
 
-    const latestPumpUpload = getLatestPumpUpload(_.get(data, 'data.upload.data', []))
-    this.source = _.get(latestPumpUpload, 'source', '').toLowerCase()
+    const pumpSettings = data.data.pumpSettings.data[0]
```

Comment:
Same question than the previous one

Reply:
In this case, the date received is from the `generateBasicsData()` method, which returns an object like this:
```ts
{
  data: {
    // ... other data
    pumpSettings: {
      data: medicalData.pumpSettings
    }
  }
}
```
And as I checked for your other comment, `medicalData.pumpSettings` is an empty array by default so I would say that there is no risk of a null pointer exception.